README.md

```diff
@@ -520,6 +520,7 @@ Simple deployment of [E-mail](https://en.wikipedia.org/wiki/Email) servers, e.g.
 - [cartulary](https://github.com/daveajones/cartulary) - RSS reader, readability tool, article archiver, microblogger, social graph manager and reading list manager. `CDDL-1.0` `PHP`
 - [Commento](https://gitlab.com/commento/commento) - Commento is a discussion platform that you can embed on your blog, news articles, and any place where you want your readers to add comments. `MIT` `Go`
 - [Coral](https://coralproject.net/) - A better commenting experience from Vox Media. ([Source Code](https://github.com/coralproject/talk)) `Apache-2.0` `Nodejs`
+- [crowd.dev](https://www.crowd.dev/) - A suite of community and data tools to unlock community-led growth for your organization. Alternative to Orbit.love, Common Room and Commsor. ([Source Code](https://github.com/CrowdDotDev/crowd.dev)) `AGPL-3.0` `Typescript`
```

Comment:
>  The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.
> https://docs.crowd.dev/docs/deployment

Docker and/or Nodejs are required, please change `Typescript` to `Docker/Nodejs`